google-cloud-examples/src/main/java/com/google/cloud/examples/bigtable/HelloWorld.java

```diff
@@ -0,0 +1,101 @@
+package com.google.cloud.examples.bigtable;
+
+import com.google.api.gax.rpc.ServerStream;
+import com.google.cloud.bigtable.admin.v2.BigtableTableAdminClient;
+import com.google.cloud.bigtable.admin.v2.BigtableTableAdminSettings;
+import com.google.cloud.bigtable.admin.v2.models.CreateTableRequest;
+import com.google.cloud.bigtable.data.v2.BigtableDataClient;
+import com.google.cloud.bigtable.data.v2.BigtableDataSettings;
+import com.google.cloud.bigtable.data.v2.models.InstanceName;
+
+import com.google.cloud.bigtable.data.v2.models.Query;
+import com.google.cloud.bigtable.data.v2.models.Row;
+import com.google.cloud.bigtable.data.v2.models.RowCell;
+import com.google.cloud.bigtable.data.v2.models.RowMutation;
+import java.sql.Timestamp;
+
+// [START HelloWorld]
+public class HelloWorld {
+
+  public static void main(String[] args) throws Exception {
+
+    // the ID of the cloud bigtable project, instance and table
+    String TABLE_ID = "Hello-Bigtable";
+    String COLUMN_FAMILY_ID = "cf1";
+    String COLUMN_QUALIFIER = "greeting";
+    String INSTANCE_ID = System.getenv("INSTANCE_ID");
+    String GCLOUD_PROJECT_ID = System.getenv("GCLOUD_PROJECT_ID");
```

Comment:
Do other product examples use environment variables? Spot checking other examples it seems like they use argument